packages/bot-strategy-runner/src/index.ts

```diff
@@ -118,13 +118,18 @@ const processExecutionOptions = async () => {
     .option("-fc, --fileConfig <path>", "input path to JSON config file.")
     .option("-uc, --urlConfig <path>", "url to JSON config hosted online. Private resources use access token")
     .option("-at, --accessToken <string>", "access token to access private configs online. EG private a repo")
+    .option("-env, --envConfig <bool>", "use a stringified config within your env set under `RUNNER_CONFIG")
```

Comment:
why do u need to set this option explicltly, why not just merge the env RUNNER_CONFIG if it exists?

Reply:
just to be more explicit.

Reply:
but you are right. I'll modify this sligtly to let it pull this in by default without the user having to say they are using an env config.

Reply:
i could imagine you might not want to delete some complex env json, but maybe disable it instead so you can easily enable it later. might be a use case for that, its up to you.